### PR TITLE
added new canonical, abstract, telemetry field

### PR DESCRIPTION
**Pull request by kevin-hereworks**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -6605,4 +6605,26 @@ VMADC:
...
+HWTTCX2:
+  description: "Hot water tank temperature control X2 with single setpoint"
+  is_abstract: true
+  opt_uses: 
+  - heater_run_command_1
+  - heater_run_command_2
+  - high_water_temperature_alarm_1
+  - high_water_temperature_alarm_2
+  uses:
+  - hot_water_tank_temperature_setpoint
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6628]: Should we also have `hot_water_tank_temperature_sensor_1` and `hot_water_tank_temperature_sensor_2`?

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 2]
```diff
@@ -313,6 +313,15 @@ HWS_SS_HWTTC_PWDPM2X:
   opt_uses:
   - compressor_run_status
 
+HWS_PWDT_HWTTCX2_SEWTM:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:316]: Change order to `HWS_SEWTM_HWTTC2X_PWDT`

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -6605,4 +6605,26 @@ VMADC:
...
+SEWTM:
+  description: "Secondary water temperature monitoring"
+  opt_uses:
+  - secondary_supply_water_temperature_sensor
+  uses:
+  - secondary_return_water_temperature_sensor
+  implements:
+  - MONITORING
+
+HWTTCX2:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6619]: Should instead by `HWTTC2X`

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -6620,3 +6620,27 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+
+SEWTM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6625]: This type already exists on line 6509. Please sync your fork with the latest master branch.

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 5]
```diff
@@ -313,6 +313,15 @@ HWS_SS_HWTTC_PWDPM2X:
...
+  description: "hot water system with 2 tanks, high water temperature alarms"
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:317]: nit: please capitalize the first word in the description.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 6]
```diff
@@ -6620,3 +6620,27 @@ VMADC:
...
+  uses:
+  - secondary_return_water_temperature_sensor
+  implements:
+  - MONITORING
+
+HWTTCX2:
+  description: "Hot water tank temperature control X2 with single setpoint"
+  is_abstract: true
+  opt_uses: 
+  - heater_run_command_1
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6638]: Hot water tanks typically do not have heating mechanisms, please explain the use of the `heater_run_command` fields or remove them from this type.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6643]: Agree, please see `HWTTC` for precedent.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 7]
```diff
@@ -3110,3 +3110,7 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- high_water_temperature_alarm:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:3114]: This field already exists, please sync your branch with the master

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Pull request closed without merging**